src/app/GitCommands/RevisionReader.cs

```diff
@@ -334,6 +336,39 @@ public void GetLog(
             }
         }
 
+        private static void AddAutoStash(IGitModule gitModule, IObserver<IReadOnlyList<GitRevision>> subject)
+        {
+            string autoStashFileName = Path.Combine(gitModule.WorkingDirGitDir, "rebase-merge/autostash");
+            if (!File.Exists(autoStashFileName)
+                || !ObjectId.TryParse(File.ReadLines(autoStashFileName).FirstOrDefault(), out ObjectId? autoStashCommitId))
+            {
+                return;
+            }
+
+            const string author = "Git";
```

Comment:
Handle as artificial commits, no info in the grid

Reply:
Except for the arbitrary author, the information in the Commit Info is valid and useful.

---

> I never felt I wanted this. Any more motivation?

It saves creating a fixup! commit and rebasing on the *previous* commit of the commit to be amended. The alternative of creating a manual stash needs more clicks, too - and can be forgotten. The changes are unreachable until the rebase is finished - a nuisance.
Just *edit* that commit (`context menu`, `a`, `e`), cherry-pick the changes from the autostash and amend.
I missed this often and, finally, noticed that git prints a commit id...
(git log `--glob=<commit id>` does not work.)

Reply:
add this to the PR description